packages/contracts-periphery/contracts/universal/op-nft/Optimist.sol

```diff
@@ -0,0 +1,44 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import "@rari-capital/solmate/src/tokens/ERC721.sol";
+import "@openzeppelin/contracts/utils/Strings.sol";
+import "./SocialContract.sol";
+
+contract Optimist is ERC721 {
+    SocialContract public sc;
```

Comment:
The social contract is never set? Also the public variables here should be immutables